src/main/js/electron/index.py

```diff
@@ -1,79 +1,97 @@
+
 from flask import Flask, jsonify, render_template, request
 app = Flask(__name__)
 import json
 import plotly
-import plotly.graph_objs as go
+import plotly.offline as plt
 
 import pandas as pd
 import numpy as np
 import pkg_resources   
 
-@app.route('/')
-def my_form():
-    return render_template("welcome.html")
+@app.route('/hello')
+def hello():
+    return render_template('basic__plot.html')
 
 @app.route('/', methods=['POST'])
 def my_form_post():
-
+    print("Hi ")
```

Comment:
please remove debug print statements like this :-)
So please push another commit to this branch to remove this line and update the PR to include that commit. Thanks!